README.md

```diff
@@ -1,23 +1,31 @@
-# PDAL Java bindings
-
 [![CI](https://github.com/PDAL/java/workflows/CI/badge.svg)](https://github.com/PDAL/java/actions) [![Join the chat at https://gitter.im/PDAL/PDAL](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PDAL/PDAL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Maven Central](https://img.shields.io/maven-central/v/io.pdal/pdal_2.13)](https://search.maven.org/search?q=g:io.pdal) [![Snapshots](https://img.shields.io/nexus/s/https/oss.sonatype.org/io.pdal/pdal_2.13)](https://oss.sonatype.org/content/repositories/snapshots/io/pdal/)
 
-Java bindings to use PDAL on JVM (supports PDAL >= 2.0).
-Mac users can experience some issues with bindings that were build against a different PDAL version,
-so try to use a consistent PDAL version.
+# PDAL Java Bindings
```

Comment:
Hmmm, not I'm thinking mb it makes sense to move it from line 3 to line 1 like its been before?